AdminClient/src/app/customers.service.ts

```diff
@@ -0,0 +1,78 @@
+import { Injectable } from '@angular/core';
```

Comment:
Can you move this and the tests to a `services/customers` path? (`AdminClient/src/app/services/customers/customers.service.ts`)